### PR TITLE
Remove all of keygen element references

### LINKED ISSUE #43: Keygen element removal
The `keygen` is stated to be in the process of being removed from the Web platform, probably due to a lack of implementations, and its use is discouraged.

So why isn't it listed as obsolete feature?

### PR DESCRIPTION
**Pull request by arronei**
- Remove all keygen definitions

Fix #43

We should probably create a Note document for this removed section and information.

### CODE REVIEW

File: sections/attributes.include [hunk 1]
```diff
@@ -512,8 +500,8 @@
     </tr>
     <tr>
       <th><code>name</code></th>
-      <td><{button}>; <{fieldset}>; <{input}>; <{keygen}>; <{output}>; <{select}>; <{textarea}></td>
-      <td>Name of form control to use for <a>form submission</a> and in the {{HTMLFormElement/elements|form.elements}} API</td>
+      <td><{button}>; <{fieldset}>; <{input}>; <{output}>; <{select}>; <{textarea}></td>
+      <td>Name of form control to use for form submission and in the {{HTMLFormElement/elements|form.elements}} API</td>
```
Comment (travisleithead) [inline sections/attributes.include]: you dropped the autolinks for `form submission`

File: sections/introduction.include [hunk 2]
```diff
@@ -460,8 +460,8 @@
   </pre>
 
   HTML documents consist of a tree of elements and text. Each element is denoted in the source by
-  a <a>start tag</a>, such as "<{body|&lt;body>}>", and an <a>end tag</a>, such as
-  "<{body|&lt;/body>}>". (Certain start tags and end tags can in certain cases be <a>omitted</a>
+  a <a>start tag</a>, such as "<{body}>", and an <a>end tag</a>, such as
+  "<{body|/body}>". (Certain start tags and end tags can in certain cases be <a>omitted</a>
```
Comment (travisleithead) [inline sections/introduction.include:464]: I think these corrections are wrong--they were supposed to include the angle-brackets..

File: sections/introduction.include [hunk 3]
```diff
@@ -880,7 +880,7 @@
       bizarre and convoluted error handling rules, user agents are permitted to fail whenever
       encountering a <a for="parser">parse error</a>.
   : Errors where the error-handling behavior is not compatible with streaming user agents
-  :: Some error-handling behavior, such as the behavior for the <code>&lt;table>&lt;hr>...</code>
+  :: Some error-handling behavior, such as the behavior for the <{table}><{hr}>...
```
Comment (travisleithead) [inline sections/introduction.include:883]: This change removes the angle-brackets which are necessary as context in the text... We can still link the name of the elements, we just need to include the angle-brackets as well.

File: sections/introduction.include [hunk 4]
```diff
@@ -1057,7 +1057,7 @@
 
       <p class="example">
         For example, if the author typed <code>&lt;capton></code> instead of
-        <code>&lt;caption></code>, this would be flagged as an error and the author could correct
+        <{caption}>, this would be flagged as an error and the author could correct
```
Comment (travisleithead) [inline sections/introduction.include:1060]: Undo this change... need the angle-brackets.

File: sections/semantics-forms.include [hunk 5]
```diff
@@ -13820,8 +13528,8 @@ control.setSelectionRange(oldStart + prefix.length, oldEnd + prefix.length, oldD
   <div class="impl">
 
   The
-  <dfn attribute for="HTMLInputElement,HTMLButtonElement,HTMLSelectElement,HTMLTextAreaElement,HTMLKeygenElement,HTMLOutputElement,HTMLFieldSetElement"><code>validity</code></dfn>
-  IDL attribute must return a {{ValidityState}} object that represents
+  <dfn attribute for="HTMLInputElement,HTMLButtonElement,HTMLSelectElement,HTMLTextAreaElement,HTMLOutputElement,HTMLFieldSetElement"><code>validity</code></dfn>
+  IDL attribute must return a <code>ValidityState</code> object that represents
```
Comment (travisleithead) [inline sections/semantics-forms.include]: Restore `{{ValidityState}}` here...

**Comment by travisleithead** [comment]
Couple of comments. Also:

(Just curious)
See if the following mentioned refs are also removed in the gererated Biblio as a result of this change:
- [[!RFC5280]]
- [[!RFC4648]]
- [[!X690]]
- [[!RFC3279]]  
- [[!RFC3447]]

I didn't notice any terms that can be removed from single_page.bs, but you might double-check to be sure.

File: sections/introduction.include (hunk 2 above)
Comment (arronei) [inline sections/introduction.include:464]: Those are correct I just built and checked them. No need to use the &lt;. The <{}> already adds the angle brackets.

File: sections/introduction.include (hunk 3 above)
Comment (arronei) [inline sections/introduction.include:883]: Same as above. Its correct.

File: sections/introduction.include (hunk 4 above)
Comment (arronei) [inline sections/introduction.include:1060]: This is correct too.

**Comment by travisleithead** [comment]
OK. I'm signing off on the technical content. Need to be sure we have consensus in the group to remove it.

**Comment by travisleithead** [comment]
As @chaals has the related issue assigned, assigning this PR to him as well.

**Comment by chaals** [comment]
We need to put this into the obsolete section as non-conforming, but I'll merge this for now.

**Comment by chaals** [comment]
Oops. I meant merge…
